crates/sui-json-rpc/src/metrics.rs

```diff
@@ -221,7 +221,7 @@ impl Logger for MetricsLogger {
             .observe(req_latency_secs);
 
         if let Some(code) = error_code {
-            if code == -32000 {
+            if code == -32000 || code == -32603 {
```

Comment:
Can we maybe add a comment here explaining what errors do -32000 and -32603 refer to.